packaging/tuned/profile/script.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+. /usr/lib/tuned/functions
+
+cpu_online_offline() {
```

Comment:
Nit: do you think any of the following names are "better"? 
1. toggle_cpu_state
2. set_cpu_availability
3. manage_cpu_online_status

Reply:
Changed, thx